### PR TITLE
export poll_latency

### PR DESCRIPTION
**Pull request by m3ngyang**
### What was wrong?

Related to Issue #1435 

### How was it fixed?
export poll_latency parameter in waitForTransactionReceipt

### Todo:
- [x] Add entry to the [release notes](https://github.com/ethereum/web3.py/blob/master/newsfragments/README.md)

#### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

**Comment by kclowes** [comment]
Thank you @m3ngyang! Will you add the new parameter to the docs when you get a chance?

**Comment by m3ngyang** [comment]
@kclowes doc test has been fixed.

File: web3/_utils/transactions.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def fill_transaction_defaults(web3, transaction):
     return merge(defaults, transaction)
 
 
-def wait_for_transaction_receipt(web3, txn_hash, timeout=120, poll_latency=0.1):
```
Comment (kclowes) [inline web3/_utils/transactions.py]: @pipermerriam is there any reason to keep the defaults here? The only place this method is used is in `eth.waitForTransactionReceipt`

**Comment by kclowes** [comment]
@m3ngyang I added a newsfragment for the release notes and cleaned up a little bit. Let me know if it doesn't look right. Otherwise, I think this is good to go!

**Comment by m3ngyang** [comment]
@kclowes the patch looks good to me.

**Pull request merged**